test/controllers/properties_controller_test.rb

```diff
@@ -46,6 +46,33 @@ class PropertiesControllerTest < ActionDispatch::IntegrationTest
     assert_enqueued_with(job: AccountSyncJob)
   end
 
+  test "handles erors when creating property" do
+    assert_no_difference [ "Account.count", "Property.count", "Account::Valuation.count", "Account::Entry.count" ] do
+      post properties_path, params: {
+        account: {
+          name: "Property",
+          currency: "USD",
+          accountable_type: "Property",
+          accountable_attributes: {
+            year_built: 2002,
+            area_value: 1000,
+            area_unit: "sqft",
+            address_attributes: {
+              line1: "123 Main St",
+              line2: "Apt 1",
+              locality: "Los Angeles",
+              region: "CA",
+              country: "US",
+              postal_code: "90001"
+            }
+          }
+        }
+      }
+      assert_redirected_to accounts_path
+      assert_equal "Balance can't be blank and Entries is invalid", flash[:alert]
+    end
+  end
```

Comment:
I think we can still remove all of these tests alongside the `rescue` handlers in the controller. 

The form validation should handle all the main requirements directly in the browser.  If a user somehow finds a way to side-step these, I'd rather the system raise an error so that we can capture it in observability systems like Sentry (rather than the user seeing an error message and the system acting as if nothing went wrong).

Following this logic, I think our existing `create` action in `accounts_controller` should probably _not_ have the `rescue` block either (as I'm sure that was the pattern you followed here).  In other words, we could probably remove this as well:

https://github.com/maybe-finance/maybe/blob/fd4011126422023228a9295b2d9b30bb1b9402cb/app/controllers/accounts_controller.rb#L61-L62